components/front.ftd

```diff
@@ -0,0 +1,165 @@
+-- component front:
```

Comment:
Please check what all container that we need to implement it. Try to keep the containers as minimum as possible.